src/components/m-dropdown/index.scss

```diff
@@ -12,16 +12,30 @@
   display: inline-block;
 }
 
+.m-dropdown--small {
+  height: $dropdown__select-height-small;
+  line-height: $dropdown__select-height-small;
+}
+
 // native select
 .m-dropdown__select-wrap {
   @include dropdown-frame();
   @include dropdown-select-wrap();
 }
 
+.m-dropdown__select-wrap--small {
```

Comment:
Won't we have just one modifier at the root node and increase CSS specificity for children?

Reply:
no please not increase specifity! Is EVIL! ahahaha. thats why we have bem. So i do not agree on this

Reply:
Well, what should I say. In general I agree with you.
Though in this case we talk about controlled specificity increase for valid reasons, like:
- styles should and can't be easily overwritten here
- all changes needed for `small` modifier can be grouped together, which increase code visibility.

Reply:
BTW. I just realized that this modifier should implemented by a reusable mixin.

Reply:
Agree for the reason that they are grouped and so easier maintainable, but still benefit is higher if we do not increase specifity especially if a consumer overrides the class. If they do it with high specificity, they have to use `!important`. So i think for the purpose of the Patterns Library is better not to do it and have a bit more classes instead. Clean Code vs Usability for Consumers

Reply:
the small modifier? so to have an iclude for the height? and one for the line-height?

Reply:
Mhm, indeed boths points are valid. In that case we should define a convention and refactor all other components if necessary